Release/include/cpprest/ws_msg.h

```diff
@@ -60,16 +60,23 @@ class websocket_outgoing_message
 public:
 #if !defined(__cplusplus_winrt)
     /// <summary>
-    /// Sets a the outgoing message to be an unsolicited pong message.
-    /// This is useful when the client side wants to check whether the server is alive.
```

Comment:
It looks like you have deleted this sentence from both `set_ping_message` and `set_pong_message`. I meant that you should keep this sentence only for `set_ping_message`, please.